client/app/globals.css

```diff
@@ -126,3 +126,8 @@ body[data-bs-theme='light'] {
     white-space: nowrap;
   }
 }
+
+/* Override Bootstrap's overflow-x for responsible tables (only in contest layouts) */
+.cc-contest-layout .table-responsive {
+  overflow-x: visible;
```

Comment:
Why is this necessary? I just tried it out, and it looks like it works fine without this, but with this style added, scrolling is disabled on mobile, which doesn't have enough width to display the entire schedule.

Reply:
If an event has a large description, the tooltip box could mess with the responsive table rendering creating a weird scroll bar.

![image](https://github.com/user-attachments/assets/352dbd66-9a2d-4100-884b-d23be15e3692)

Reply:
![image](https://github.com/user-attachments/assets/9f0a1aa2-7520-4a9c-b3e3-538821f8a6c7)

Reply:
Ohhh, good catch! Shouldn't it be overflow-y: visible in that case though? Like I said, this disables horizontal scrolling on mobile.

Reply:
I experimented with `overflow-y` but it didnt seem to do anything. we might need to redesign the tooltip and/or table design to get around this, or may just need to use the default bootstrap tooltips. Or perhaps make the tooltips size 0 until the button is hovered.

Reply:
The Bootstrap tooltips require JS, but we just use the Bootstrap CSS classes. I'll try to play around with this a bit.

Reply:
I'll just look into it separately, I just realized there's no reason to be holding up this PR over this